start_dev.sh

```
@@ -0,0 +1,29 @@
+#!/bin/bash
+echo "=== Starting Unified Development Environment ==="
+echo "🌐 Everything through: http://localhost:3000"
+echo
+
+echo "🧹 Cleaning up previous containers..."
+cd k8s
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)